lambdas/utils/lloyd_george_validator.py

```diff
@@ -167,7 +173,7 @@ def validate_with_pds_service(file_name_list: list[str], nhs_number: str):
         if patient_details.general_practice_ods != current_user_ods:
             raise LGInvalidFilesException("User is not allowed to access patient")
 
-    except (ValidationError, ClientError, ValueError) as e:
+    except (ClientError, ValueError) as e:
```

Comment:
Would it be safer to keep ValidationError in this catch block?  
Although the chance is slim, but I think ValidationError may occur at line 156 when we try to parse PDS's response